src/constants/index.ts

```diff
@@ -16,6 +16,14 @@ export const USDT = new Token(ChainId.MAINNET, '0xdAC17F958D2ee523a2206206994597
 export const COMP = new Token(ChainId.MAINNET, '0xc00e94Cb662C3520282E6f5717214004A7f26888', 18, 'COMP', 'Compound')
 export const MKR = new Token(ChainId.MAINNET, '0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2', 18, 'MKR', 'Maker')
 export const AMPL = new Token(ChainId.MAINNET, '0xD46bA6D942050d489DBd938a2C909A5d5039A161', 9, 'AMPL', 'Ampleforth')
+export const SUSD = new Token(
+  ChainId.MAINNET,
+  '0x57Ab1ec28D129707052df4dF418D58a2D46d5f51',
+  18,
+  'sUSD',
+  'Synthetix sUSD'
+)
+export const BASED = new Token(ChainId.MAINNET, '0x68A118Ef45063051Eac49c7e647CE5Ace48a68a5', 18, '$BASED', '$BASED')
```

Comment:
can you send a link verifying this token's identity? also, mind briefly explaining the need for the special-case routing?

Reply:
refs:
- https://stake.based.money/
- https://etherscan.io/token/0x68a118ef45063051eac49c7e647ce5ace48a68a5
- https://uniswap.info/token/0x68a118ef45063051eac49c7e647ce5ace48a68a5
- https://uniswap.info/pair/0xaAD22f5543FCDaA694B68f94Be177B561836AE57

rationale:
- special-case routing is necessary because the project has chosen to incentivize Uniswap LPs of the sUSD:$BASED pair.
- sUSD is not on the routing shortlist (`BASES_TO_TRADE_AGAINST`) and only currently has significant liquidity with ETH and $BASED, so it does not make sense to add it to the routing shortlist for all tokens.
- the special-case routing ensures that Uniswap correctly routefinds through sUSD, allowing trades between ETH (and everything liquid with ETH) through sUSD to $BASED.